Types.ts

```diff
@@ -291,7 +291,7 @@ export interface LinkError {
     errorCode: LinkErrorCode;
     errorType: LinkErrorType;
     errorMessage: string;
-    errorDisplayMessage?: string;
+    displayMessage?: string;
```

Comment:
If I'm understanding the issue here, it's that Android uses `displayMessage` and iOS uses `errorDisplayMessage`.

`errorDisplayMessage` appears to be the correct name as that's what's exposed in `Types.ts` – let's change Android to pass back `errorDisplayMessage` instead of changing the public API and ObjC to match.

@amytang0

Reply:
@pladaub hi, sorry for the delay here - could you let me know the best way to update Android to pass back `errorDisplayMessage` instead of `displayMessage`?

Reply:
Hey Avery, thanks for opening the PR. We fixed that displayMessage here in this PR: https://github.com/plaid/react-native-plaid-link-sdk/commit/47f6fb2855bd28d8a3d2067b97bafc6a5d8fc433